lambdas/service/app.py

```diff
@@ -86,7 +86,77 @@
 
 openapi_spec = {
     'info': {
-        'description': 'This should probably be really long',
+        'description': format_description('''
+            # Overview
+            Azul provides a programmatic interface for quickly and uniformly
+            querying metadata associated with experimental and analysis data
+            stored in the
+            [data store (DSS)](https://github.com/HumanCellAtlas/data-store).
+            Azul serves as the backend for the
+            [data-browser](https://github.com/DataBiosphere/data-browser).
+
+            Azul aggregates metadata into several different indices:
+             - [files](#operations-Repository-get_repository_files)
+             - [samples](#operations-Repository-get_repository_samples)
+             - [bundles](#operations-Repository-get_repository_bundles)
+
+            which are all stored in respective Elasticsearch indices.
+
+            Azul provides the ability to download relevant data via the
+            [Manifests](#/Manifests) endpoints. After refining a search with the
```

Comment:
This is misleading. There is no requirement to hit the repo endpoints first. Try to rewrite this,.